package com.desafio.meetime.infrastructure.exception

class ContactSaveException(message: String?, cause: Throwable? = null) : RuntimeException(message, cause)
